bodyfetcher.py

```diff
@@ -124,9 +134,16 @@ def make_api_call_for_site(self, site):
                 pagesize = "50"
             else:
                 pagesize = "25"
-            url = "http://api.stackexchange.com/2.2/questions?site=stackoverflow&filter=!)E0g*ODaEZ(SgULQhYvCYbu09*ss(bKFdnTrGmGUxnqPptuHP&key=IAkbitmze4B8KpacUfLqkw((&pagesize=" + pagesize + min_query
+            pagesize_modifier = "&pagesize={0}{1}".format(pagesize, min_query)
         else:
-            url = "http://api.stackexchange.com/2.2/questions/" + ";".join(str(x) for x in posts) + "?site=" + site + "&filter=!)E0g*ODaEZ(SgULQhYvCYbu09*ss(bKFdnTrGmGUxnqPptuHP&key=IAkbitmze4B8KpacUfLqkw(("
+            question_modifier = "/{0}".format(";".join(str(x) for x in posts))
```

Comment:
There was also a formatting issue here somewhere that I think this fixed... although I'm not 100% that it fixed it <_<